pom.xml

```diff
@@ -160,7 +160,8 @@ flexible messaging model and an intuitive client API.</description>
     <rocksdb.version>5.13.3</rocksdb.version>
     <slf4j.version>1.7.25</slf4j.version>
     <commons.collections.version>3.2.2</commons.collections.version>
-    <log4j2.version>2.10.0</log4j2.version>
+    <log4j2.version>2.11.2</log4j2.version>
```

Comment:
Is this upgrade required by this PR? In any case, I'd leave it separate

Reply:
There is a bug in previous version